skietbaan_fe/src/scss/view-comp.css

```diff
@@ -78,12 +110,78 @@
 }
 
 .td-col{
+    vertical-align: top;
 	padding-top: 11px;
 	padding-bottom: 11px;
-	text-align: left;
+    width: 90%;/*don't forget to remove*/
 }
 
 .page-contents {
 	background-color: transparent;
 	padding-top: 60px;
 }
+
+.test1{
+    float: left;
+    width: 80%;
+    vertical-align: top;
+}
+
+.test2{
+    float: left;
+    width: 20%;
+    vertical-align: top;
+}
+
+.test3{
+    float: left;
+    width: 10%;
+    vertical-align: top;
+}
+
+.Collapsible__contentOuter{
+    float: left;
+}
+
+
+.comp-req-container {
+	margin: 20px;
+	padding: 20px;
+	text-align: center;
+}
+
+.req-input {	
```

Comment:
try using prettier so code is formatted properly, seems like most spacing is inconsistent

Reply:
Yes, or just indent like normally.